config/md5

```diff
@@ -583,8 +583,22 @@ https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/14.10/x86_64/ruby-2.1.5.tar
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/14.10/x86_64/ruby-2.2.0.tar.bz2=e68c9d66cf6309e05056b8e381a9f05c
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/14.10/x86_64/ruby-2.2.1.tar.bz2=30ddde33731e10c38e4857384edca8f2
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.0.0-p648.tar.bz2=7c1d6de9b87613abcb9e21fff871352c
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.1.0.tar.bz2=9379ac4dfde8087ddf0cdb5015274e6b
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.1.1.tar.bz2=f565fc095825c45b5a19abfff5609d56
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.1.2.tar.bz2=52fbd7251723038c6570db27d0c56afe
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.1.3.tar.bz2=22cbde066f7c8686837d4e4e214bf4df
+https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.1.4.tar.bz2=e0ff2e068b376f0db37355954b91468c
 https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.1.5.tar.bz2=f470ce5b7bce1230a3e35707cc88a473
-https://rvm_io.global.ssl.fastly.net/binaries/ubuntu/16.04/x86_64/ruby-2.1.9.tar.bz2=1c9787ca23ab930330fb2c24bafa3330
```

Comment:
Hmm, why would the built hash be different?

Reply:
> Ruby 2.1.9 was replaced with new distribution (by mistake). Md5 and sha updated with new links